BBBadgeBarButtonItem/BBBadgeBarButtonItem.m

```diff
@@ -142,6 +141,7 @@ - (void)setBadgeValue:(NSString *)badgeValue
         self.badge.backgroundColor      = self.badgeBGColor;
         self.badge.font                 = self.badgeFont;
         self.badge.textAlignment        = NSTextAlignmentCenter;
+        self.badge.layer.masksToBounds  = YES;
 
```

Comment:
Thank you julian, I already know about this fix but I don't get on on my pod install/update ... I working with multiple private and public pods and I need to update them often so I cannot just add it to the .m file because it will undo the change each time I update pods. When is this line of code going to be added on the pod?